tests/en/test_nsw_art_union.catala_en.result

```diff
@@ -10,7 +10,9 @@ declaration scope Test1:
   my_gaming scope GamingAuthorized
 
 scope Test1:
-  definition my_gaming.artUnion equals
+  definition
```

Comment:
it should not split `definition my_gaming.artUnion equals` into three lines if `definition my_gaming.artUnion equals` does not overflow over 80 characters of wideness

Reply:
It depends on the input, in this particular case the input was :
```
scope Test1:
  definition
    my_gaming.artUnion
  equals
    ArtUnion {
      -- grossProceeds: $50000
      -- typeOrg: NonProfit
      -- proceedsToBenefitingOrg: $25000
      -- totalValueOfThePrizes: $45000
      -- maxAmountMoneyAsSeparatePrize: $100
      -- holdsAuthority: true
    }
  assertion my_gaming.authorized = true
```
Hence, there is a separation happening.

More generally, if `( definition ... equals )` is on several lines, then it splits in three.

Reply:
That's exactly what it does actually, it looks at [definition x equals] as one specific block checks whether it contains some line-breaks and split in three or leave it as a one-liner.
For instance:
```
définition x égal à 1+ 2
définition y égal à
1+ 2
```
stays together or splits after the "égal à"
```
définition x égal à 1 + 2
définition x égal à 
  1 + 2
```

What you would want require topiary to be able to determine if it would fit or not. And that's the main issue, it's not able to do that so it relies on the existing user line breaks.

Reply:
OK we're on the same page. What I still don't understand is why this PR splits the line `definition my_gaming.artUnion equals` at the place where I tagged it. It was on a single line before, thus it should remain on a single line? Or have you manually inserted line breaks at these places?

Reply:
Ah ok, I get your misunderstanding.
The original input was what I quoted:
```
  definition
    my_gaming.artUnion
  equals
    ...
```
The previously committed input was:
``` definition my_gaming.artUnion equals
    ...
```
BUT the changes contains rules that consider break-lines in such constructions and therefore the new committed result is with line-breaks which is similar to the original.

Sorry for the quid pro quo, it seemed obvious to me.